plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/AbstractExecutionContext.java

```diff
@@ -119,13 +121,22 @@ protected boolean initContextBootstrap(@NotNull DBRProgressMonitor monitor, bool
                             throw new DBCException(message, e, this);
                         }
                     }
+                    monitor.worked(1);
                 }
             }
             return true;
         }
         return false;
     }
 
+    /**
+     * Returns a list of queries that should be performed before initializing the session.
+     */
+    @NotNull
+    public List<String> getExtraInitQueries() {
+        return new ArrayList<>();
```

Comment:
```suggestion
        return List.of();
```

Reply:
* Returns an unmodifiable list containing zero elements.

I do not think so.

Reply:
Maybe you'd prefer `Collections.emptyList()`?

Reply:
Both `List.of()` and `Collections.emptyList()` returns a static instance of an empty list, resulting in no additional allocations.

![image](https://user-images.githubusercontent.com/35821147/209808951-e9d814a1-9ddc-4bb2-b566-97881ca98b8e.png)